src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/ActivityHelper.cs

```diff
@@ -62,7 +62,13 @@ public static bool HasStarted(HttpContext context, out Activity? aspNetActivity)
     {
         PropagationContext propagationContext = textMapPropagator.Extract(default, context.Request, HttpRequestHeaderValuesGetter);
 
-        Activity? activity = AspNetSource.StartActivity(TelemetryHttpModule.AspNetActivityName, ActivityKind.Server, propagationContext.ActivityContext);
+        IEnumerable<KeyValuePair<string, object?>>? enumerable = null;
+        if (context.Request?.Unvalidated?.Path is string path)
+        {
+            enumerable = [new KeyValuePair<string, object?>("url.path", path)];
```

Comment:
is this tag also populated by the aspnet instrumentation as well?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AspNet/Implementation/HttpInListener.cs#L87C67-L87C80 can be removed, if we proceed with this approach.

Reply:
It can only be removed if it is required that TelemetryHttpModule be used with AspNet. It's possible to use AspNet without the telemetry module and in that case users would be missing the `url.path` attribute.

Reply:
> if it is required that TelemetryHttpModule be used with AspNet.

It is [required!](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet#step-2-modify-webconfig)

Reply:
Ok. removed from AspNet because it's now set in the telemetry module. That was the only unvalidated property being accessed.